build/build.sh

```diff
@@ -69,12 +69,13 @@ moveCompiledBuildx(){
 buildx(){
     output_name="bin/$CTLNAME"
     echo "Building for: ${GOOS} ${GOARCH}"
-    gox -cgo -os="$GOOS" -arch="$GOARCH" -ldflags \
-        "-X github.com/openebs/node-disk-manager/pkg/version.GitCommit=${GIT_COMMIT} \
-        -X main.CtlName='${CTLNAME}' \
-        -X github.com/openebs/node-disk-manager/pkg/version.Version=${VERSION}" \
-        -output="$output_name" \
-        ./cmd/"$BUILDPATH"
+     CGO_ENABLED=1 GOOS=${GOOS} GOARCH=${GOARCH} \
```

Comment:
Its not required to set `GOOS` and `GOARCH`, if we are using `go build` as it automatically picks it up from the `go env`

Reply:
Also, CGO_ENABLED env can be exported. So the build command becomes cleaner,